### ISSUE TITLE
Game doesn't build under Visual Studio 2013

### ISSUE THREAD

**Issue by willglynn** [raceintospace/raceintospace#128]
This is blocked by [protobuf issue #531](https://code.google.com/p/protobuf/issues/detail?id=531) which was already fixed upstream.

**Comment by sflynn850** [comment]
Should we consider closing this issue? This ticket is gonna have it's 11th birthday, and VS2013 is probably not in high demand.

**Comment by peyre** [comment]
Yeah, maybe so.  I want, at some point, to update the Windows compiling instructions in the README, but there's a separate Issue for that already.
